Exercise_7/scripts/ticTacToe/KeyActions.js

```diff
@@ -0,0 +1,42 @@
+import {SPACE_KEY, LEFT_KEY, RIGHT_KEY, UP_KEY, DOWN_KEY} from './Constraints';
```

Comment:
Please, add spaces in the beginning of the literal and at the end of it.
https://github.com/airbnb/javascript/blob/master/README.md#whitespace--in-braces